.github/workflows/pr-open.yml

```diff
@@ -161,12 +161,53 @@ jobs:
       DOMAIN: apps.silver.devops.gov.bc.ca
     steps:
       - uses: actions/checkout@v3
+
+      - name: Vault Broker
+        id: broker
+        run: |
+          # Read the intention file and replace the event url and the user id
```

Comment:
Please address coding consistency, like trailing right brackets.  Some lines have a break before, some don't.  There is also inconsistency in where lines are broken and length, like INTENTION vs VAULT_TOKEN_WRAP.